alias_service.py

```diff
@@ -0,0 +1,25 @@
+from model import Player
+
+def get_top_suggestion_for_aliases(dao, aliases):
```

Comment:
This is probably the function I would call from the POST handler right? Does it return a map from alias -> ID? Just remember that ID can also be None if it doesn't match.